app/res/layout/activity_drawing_boundary.xml

```diff
@@ -44,32 +44,32 @@
         android:layout_width="match_parent"
         android:orientation="horizontal">
 
-        <Button
-            style="@style/MapButtonGreen"
+        <com.google.android.material.button.MaterialButton
+            android:backgroundTint="@color/green_500"
```

Comment:
The old styled defined a selector that changes based on pressed button state, any reason we are no longer doing that ?

Reply:
This is updated in the last commit i have created the new style for this which is of same behaviour as the previous one

Reply:
can you please link me to the style you are referring to ?